packages/contracts/docs/core/02-how-to-guides/01-plugin-development/03-publication/02-versioning.md

```diff
@@ -1,29 +1,48 @@
 ---
-title: Version Numbers
+title: Versioning
 ---
 
-## Using Semantic Versioning for Your Plugin
+## Versioning Your Plugin
 
-:::note
-Work in Progress
-:::
+The aragonOSx plugin ecosystem has an on-chain versioning system built-in.
 
-<!--TODO: This needs a core team discussion-->
+Given a version tag `RELEASE.BUILD`, we can infer that:
 
-Our plugin version numbering follows the common [semantic versioning](https://semver.org/) notation. Below, we define what a version bumps means in the context of the `Plugin` ecosystem.
+1.  We are doing a `RELEASE` version when we apply breaking changes affecting the interaction with other contracts on the blockchain to:
```

Comment:
Is this section a draft for future continuation?

Reply:
Yes, the intention behind this page is to give the basic rules for assigning the version number. 
It misses an explanation that the build number is strictly incremented and that you always can just install the latest build of release. I am not sure where to put this explanation. 

I think it is good to have this versioning guideline (in analogy to the semver convention) and fixed graphics related to versioning now already so that the auditors are not confused by the old documentation.

Reply:
In case the formatting appears odd to you:

This is how it looks https://github.com/aragon/core/blob/b27c7c596298fd51920df929b41da4f86b453b68/packages/contracts/docs/core/02-how-to-guides/01-plugin-development/03-publication/02-versioning.md